app/experimenter/nimbus-ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Can we fold this into the project wide `.gitignore`? 

https://github.com/mozilla/experimenter/blob/main/.gitignore

Reply:
Yeah! I left a comment at the top of this file mentioning I meant to remove this, will do with next push. :D